android/envoy/src/main/kotlin/org/greatfire/envoy/NetworkIntentService.kt

```diff
@@ -188,15 +139,15 @@ class NetworkIntentService : IntentService("NetworkIntentService") {
     }
 
     private fun broadcastUpdateFailed(url: String) {
-        Log.e(TAG, "broadcast failure for url: " + url)
+        Log.e(TAG, "broadcast update failure for url: " + UrlUtil.sanitizeUrl(url, ENVOY_SERVICE_UPDATE))
```

Comment:
maybe you should put the sanitized url into a val so you don't have to run it every time

Reply:
i went through the code and added a val wherever i made duplicate calls, but here i need to pass in the url for the intent and only call the method once so i'm not sure a val makes sense